packages/core-data/src/actions.js

```diff
@@ -126,14 +126,15 @@ export function receiveEmbedPreview( url, preview ) {
  * Returns an action object that triggers an
  * edit to an entity record.
  *
- * @param {string} kind           Kind of the edited entity record.
- * @param {string} name           Name of the edited entity record.
- * @param {number} recordId       Record ID of the edited entity record.
- * @param {Object} edits          The edits.
+ * @param {string}  kind       Kind of the edited entity record.
+ * @param {string}  name       Name of the edited entity record.
+ * @param {number}  recordId   Record ID of the edited entity record.
+ * @param {Object}  edits      The edits.
+ * @param {boolean} undoIgnore Whether to ignore the edit in undo history or not.
  *
  * @return {Object} Action object.
  */
-export function* editEntityRecord( kind, name, recordId, edits ) {
+export function* editEntityRecord( kind, name, recordId, edits, undoIgnore ) {
```

Comment:
Should we mark this argument as experimental? Also maybe an object is better in case we'd need more "options" in the future?

Reply:
I don't think it should be experimental, but it should be an object.

Reply:
https://github.com/WordPress/gutenberg/pull/17452/commits/6013792016eae435feeb65cd4992220faba8fa14